specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/stable/2025-03-01/fleets.json

```diff
@@ -852,6 +852,7 @@
             "headers": {
               "Azure-AsyncOperation": {
                 "type": "string",
+                "format": "uri",
```

Comment:
Removed the ARM breaking change detection since this is not real REST breakings.

Reply:
remove won't work

you need to add label like -approved-bugfix or -approved-benigh